importer/tasks.py

```diff
@@ -0,0 +1,11 @@
+"""
```

Comment:
It's a good idea to have in general, and it's only not there because I forgot it. There are no strings in this file, though, so it doesn't matter.